opensearch-project/OpenSearch-Dashboards
# [Sample Data] Update demo dashboard title

### Description

Updates the sample dashboard title and description that shows the various visualizations to better reflect what it is.

### Issues Resolved




## Screenshot


<img width="1001" alt="Screenshot 2024-07-12 at 11 52 17 AM" src="https://github.com/user-attachments/assets/1c3417a0-1b82-4f26-93ea-dcaf55f9f555">

## Testing the changes


1. Add web logs sample data
2. Navigate to Dashboard listing page
3. See the new title for the demo dashboard

## Changelog

- fix: [Sample Data] Updates sample dashboard title in sample web logs data

### Check List
